robot_smach_states/src/robot_smach_states/startup.py

```diff
@@ -239,3 +239,44 @@ def execute(self, userdata=None):
 
         rospy.loginfo("Timed out with door still closed")
         return "closed"
+
+
+class WaitForLocalPlanner(smach.State):
+    def __init__(self, robot, timeout):
```

Comment:
Docstrings missing ;-). Especially important for robot skills and robot smach states: if someone wants to use it, he/she shouldn't have to look at the code